lib/matplotlib/tests/test_contour.py

```diff
@@ -158,7 +158,7 @@ def test_contour_manual_labels():
 
 @image_comparison(baseline_images=['contour_labels_size_color'],
                   extensions=['png'], remove_text=True)
-def test_contour_manual_labels():
+def test_contour_labels_size_color():
 
```

Comment:
Travis is failing on this image comparison. I'm guessing this just needs an updated image, since it's been a while since this test was ever run.